src/ice_usb_uart.c

```diff
@@ -0,0 +1,245 @@
+//
+// Created by MrJake222 on 03/11/24.
+//
```

Comment:
Would it be possible to make this a copyright statement?

If you wanted to avoid the complete MIT license block and use C++ style comments:

```suggestion
//
// Copyright (c) 2024 MrJake222
// SPDX-License-Identifier: MIT
//
```